### PR TITLE
Update nf-core to 2.8

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`nf-core`](https://bioconda.github.io/recipes/nf-core/README.html): **2.7.2** &rarr; **2.8**

[install with bioconda](http://bioconda.github.io/recipes/nf-core/README.html) [Conda](https://anaconda.org/bioconda/nf-core/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/nf-core`](https://github.com//bioconda/bioconda-recipes/tree/bump/nf_core/recipes/nf-core) (click to view/edit other files)
Summary | Python package with helper tools for the nf-core community.
Home | [http://nf-co.re/](http://nf-co.re/)
Releases |[https://pypi.org/pypi/nf-core/json](https://pypi.org/pypi/nf-core/json)

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by jfy133** [comment]
@mirpedrol @mashehu did I miss anything else in terms of new or dependency version bumps?

File: recipes/nf-core/meta.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ requirements:
     - markdown >=3.3
     - jsonschema >=3.0
     - jinja2
-    - rich >=10.7.0
+    - rich >=13.3.1
     - rich-click >=1.0.0
```
Comment (mashehu) [inline recipes/nf-core/meta.yaml:36]: `rich>=13.3.1`

Comment (mirpedrol) [reply recipes/nf-core/meta.yaml:36]: and `rich-click>=1.6.1`

Comment (jfy133) [inline recipes/nf-core/meta.yaml:36]:
```suggestion
     - rich-click >=1.6.1
```

**Review by jfy133 — APPROVED** [review]

**Pull request merged**